decentraland/unity-explorer
# feat: waves effect on loading screen background

Adds the color configuration to the default tips, so its used for interpolating the background color of the loading screen with the new waves effect.

https://github.com/decentraland/unity-explorer/assets/56365551/c5c9a9e2-6a85-4d0f-874d-b0e4e88391c9

# How to test?

1. Go through the authentication process
2. Jump into world
3. Observe the new background effect while the world is loading
4. Change the tips with the arrows/breadcrumbs
5. Teleport to other locations, loading screen should keep working as expected
